src/main/java/org/broadinstitute/hellbender/tools/picard/analysis/artifacts/CollectSequencingArtifactMetrics.java

```diff
@@ -51,6 +52,10 @@
 public final class CollectSequencingArtifactMetrics extends SinglePassSamProgram {
     static final String USAGE = "Produces metrics to quantify single-base sequencing artifacts from a SAM/BAM file";
 
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc = "File to write the output to.")
+    public File OUTPUT;
```

Comment:
Because the collectors need be instantiable in multiple contexts, so all of their args need to be in their own argument collection, and available in all contexts. If I leave it in SinglePassSam, it will be a duplicate arg in that context. As the other collectors are moved to this pattern, the individual ones that were moved down will go away and there will only be one again (in the MetricsArgumentCollection base class).